client/network/src/protocol/generic_proto/handler/group.rs

```diff
@@ -693,9 +700,9 @@ impl ProtocolsHandler for NotifsHandler {
 						},
 					ProtocolsHandlerEvent::Custom(NotifsInHandlerOut::Closed) => {},
 					ProtocolsHandlerEvent::Custom(NotifsInHandlerOut::Notif(message)) => {
```

Comment:
Do I understand correctly that this `message` is dropped in case not all substreams have been either negotiated or refused yet? While not a change specific to this pull request, if that is the case, would you mind adding a short comment?

Reply:
This should only ever be reached if the local node has disconnected from a peer, but this peer is making itself comfortable and is continuing to send notifications.